granulate_utils/config_feeder/client/bigdata/emr.py

```diff
@@ -0,0 +1,40 @@
+import json
+from typing import Dict, Optional
+
+from granulate_utils.config_feeder.core.models.cluster import BigDataPlatform, CloudProvider
+from granulate_utils.config_feeder.core.models.node import NodeInfo
+
+
+def get_emr_node_info() -> Optional[NodeInfo]:
```

Comment:
Please provide EMR versions that this was tested on, let's test just 2 "edge" versions, for example 5.2x and the newest 6.x EMR release versions

Reply:
tested on 5.20.1 and 6.11.0